src/Dfe.PlanTech.Infrastructure.Data/CmsDbContext.cs

```diff
@@ -249,17 +247,13 @@ private static string GetCacheKey(IQueryable query)
                 .AsSplitQuery(),
             cancellationToken);
 
-    public async Task<List<T>> ToListAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default)
+    public Task<List<T>> ToListAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default)
     {
-        var key = GetCacheKey(queryable);
-        return await _queryCacher.GetOrCreateAsyncWithCache(key, queryable,
-            (q, ctoken) => q.ToListAsync(ctoken), cancellationToken);
+        return queryable.ToListAsync(cancellationToken);
     }
 
-    public async Task<T?> FirstOrDefaultAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default)
+    public Task<T?> FirstOrDefaultAsync<T>(IQueryable<T> queryable, CancellationToken cancellationToken = default)
     {
-        var key = GetCacheKey(queryable);
-        return await _queryCacher.GetOrCreateAsyncWithCache(key, queryable,
-            (q, ctoken) => q.FirstOrDefaultAsync(ctoken), cancellationToken);
+        return queryable.FirstOrDefaultAsync(cancellationToken);
```

Comment:
why'd all this get reverted?

Reply:
Debugging that I left in accidentally, whoops. Back in now